sections/k-two-col-carousel.liquid

```diff
@@ -1,55 +1,242 @@
 <link rel="stylesheet" href="{{ 'k-two-col-carousel-styles.css' | asset_url }}" media="print" onload="this.media='all'">
```

Comment:
What about this stylesheet? It seems like it (mostly?) won't be needed anymore? or do these styles still apply?

Reply:
no, we need it, applied new css codes on that file

Reply:
makin new commit on this branch

Reply:
@heythisisnate made a new commit and edits on this branch. Please check